src/coord/src/catalog/builtin.rs

```diff
@@ -671,6 +671,22 @@ JOIN mz_catalog.mz_objects ON mz_indexes.on_global_id = mz_objects.global_id",
     needs_logs: false,
 };
 
+pub const PG_DESCRIPTION: BuiltinView = BuiltinView {
+    name: "pg_description",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_description AS SELECT
+    oid as objoid,
+    NULL::oid as classoid,
+    CASE
+        WHEN relkind = 'i' THEN NULL::int4
+        ELSE 0::int4
+    END objsubid,
```

Comment:
Any particular reason to special case indexes as null? Based on my read of `pg_description` it seems like it's 0 for indexes? But could definitely be different in practice, or I could be misunderstanding!

Reply:
No you're right! Switching that now.